PATCH-P0-FIX-SUSE: fix ppc64 libdir

This patch fixes libdir on ppc64

Maintainer: Dinar Valeev <dvaleev@suse.com>

Index: cmake/install_layout.cmake
===================================================================
--- cmake/install_layout.cmake.orig
+++ cmake/install_layout.cmake
@@ -138,7 +138,7 @@
 SET(INSTALL_SBINDIR_RPM                 "sbin")
 SET(INSTALL_SCRIPTDIR_RPM               "bin")
 #
-IF(CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64")
+IF(CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64" "OR" CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64")
   SET(INSTALL_LIBDIR_RPM                "lib64")
   SET(INSTALL_PLUGINDIR_RPM             "lib64/mysql/plugin")
 ELSE()
